main-process/native-ui/tray/tray.js

```diff
@@ -15,17 +15,25 @@ ipc.on('put-in-tray', function (event) {
     label: 'Remove',
     click: function () {
       event.sender.send('tray-removed')
-      appIcon.destroy()
```

Comment:
I think the crash only happens when called in a click handler right?

So perhaps calling `ipc.send('remove-tray')` after this line would make it so we don't even need to destroy it from this click handler https://github.com/electron/electron-api-demos/blob/8d71761cb0eae35034c8649dd5f24595bbee870b/renderer-process/native-ui/tray/tray.js#L21